include/dpp/user.h

```diff
@@ -93,6 +93,8 @@ class DPP_EXPORT user : public managed, public json_interface<user>  {
 	std::string global_name;
 	/** Avatar hash */
 	utility::iconhash avatar;
+	/** Avatar decoration hash */
+	utility::iconhash avatar_decoration;
 	/** Flags built from a bitmask of values in dpp::user_flags */
```

Comment:
should this be here or under user_identified?
Are the avatar_decoration sent via the websocket?

Reply:
that's a good question. Idk. can't test that either as it's not out yet 😥